.github/workflows/upload-algolia-api.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import annotations  # noqa: INP001
```

Comment:
This honestly isn't that horrid of a script. A few more comments/docstrings to explain the method to the method madness would help though. "This script generates records for algolia to search for all methods/functions because ...., the records look like ...."

Reply:
Tried to document a bit more.  Time to :ship: